spring-integration-core/src/test/java/org/springframework/integration/dsl/routers/RouterTests.java

```diff
@@ -936,15 +952,19 @@ public IntegrationFlow propagateErrorFromGatherer(TaskExecutor taskExecutor) {
 					.get();
 		}
 
+		@Bean
+		public PollableChannel scatterGatherWiretapChannel() {
+			return new QueueChannel();
+		}
```

Comment:
We need a new line in between these methods.

Reply:
fixed.

Reply:
No reason to comment if you agree with my suggestion for a fix.
I will see your change in the next commit.

I'd like to see comments on review only if you are not agreed with me.